seqseek/chromosome.py

```diff
@@ -45,14 +46,18 @@ def validate_name(self):
         if self.name not in self.chromosome_lengths.keys():
             raise ValueError("{name} is not a valid chromosome name".format(name=self.name))
 
-    def validate_coordinates(self, start, end):
-        if start < 0 or end < 0:
-            raise ValueError("Start and end must be positive integers")
+    def validate_coordinates(self, start, end, loop=False):
+        if loop and self.name != 'MT':
+            raise ValueError('Loop may only be specified for the mitochondria.')
+        if (start < 0 and not loop) or end < 0:
+            raise ValueError("Start and end must be positive integers for this chromosome")
         if end < start:
             raise ValueError("Start position cannot be greater than end position")
-        if start > self.length or end > self.length:
+        if start > self.length or (end > self.length and not loop):
             raise ValueError('Coordinates out of bounds. Chr {} has {} bases.'.format(
                 self.name, self.length))
+        if loop and end - start > self.length:
+            raise TooManyLoops()
```

Comment:
The name of this exception makes me think of https://www.youtube.com/watch?v=QrGrOK8oZG8 and I love it.